vcd/datasource_vcd_ip_space.go

```diff
@@ -124,6 +124,21 @@ func datasourceVcdIpSpace() *schema.Resource {
 				Computed:    true,
 				Description: "Flag exposing if route advertisement is enabled",
 			},
+			"default_firewall_rule_creation_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Flag whether default firewall rule creation should be enabled (VCD 10.5.0+)",
```

Comment:
I'm kinda missing the verb there 😄 
A suggestion, copied from the attribute from above:
```suggestion
				Description: "Flag exposing whether default firewall rule creation should be enabled (VCD 10.5.0+)",
```
This would apply also to the other two below and the resource ones.

Reply:
Changed.